packages/shared/src/graphql/posts.ts

```diff
@@ -692,15 +683,41 @@ export const createPost = async (
   return res.createFreeformPost;
 };
 
-export const createSourcePostModeration = async (
-  variables: Partial<CreatePostModerationProps>,
-): Promise<SourcePostModeration> => {
-  const res = await gqlClient.request(
-    CREATE_SOURCE_POST_MODERATION_MUTATION,
-    variables,
-  );
+export const SOURCE_POST_MODERATION_QUERY = gql`
+  query SourcePostModeration($id: ID!) {
```

Comment:
I cannot test this feature because I get error on this query because `sourceId` is required parameter on server query, do I need to use a specific branch on BE? 
<img width="548" alt="image" src="https://github.com/user-attachments/assets/f3429e3a-8922-4884-9225-86e049b4b891">

Reply:
This should be merged together with the API PR. Only then you can test it. For now you can check it locally together.

Reply:
Because if we merge the API alone, some parts of the current preview will break.